### PR TITLE
Bento docs: split Example and Import via script sections

### PR DESCRIPTION
**Pull request by sebastianbenz**

### CODE REVIEW

File: extensions/amp-accordion/1.0/README.md [hunk 1]
```diff
@@ -25,7 +25,17 @@ import {defineElement as defineBentoAccordion} from '@bentoproject/accordion';
...
 
-### Example: Include via `<script>`
+### Include via `<script>`
+
+```html
+<script type="module" src="https://cdn.ampproject.org/bento.mjs"></script>
+<script type="nomodule" src="https://cdn.ampproject.org/bento.js"></script>
+<script type="module" src="https://cdn.ampproject.org/v0/bento-accordion-1.0.mjs"></script>
+<script type="nomodule"  src="https://cdn.ampproject.org/v0/bento-accordion-1.0.js"></script>
+<link rel="stylesheet" type="text/css" href="https://cdn.ampproject.org/v0/bento-accordion-1.0.css"/>
```
Comment (westonruter) [inline extensions/amp-accordion/1.0/README.md:35]: Any reason for using self-closing tag for `link` elements?

```suggestion
<link rel="stylesheet" type="text/css" href="https://cdn.ampproject.org/v0/bento-accordion-1.0.css">
```

It's only relevant for XHTML which is largely from a bygone era.

File: extensions/amp-accordion/1.0/README.md [hunk 2]
```diff
@@ -25,7 +25,17 @@ import {defineElement as defineBentoAccordion} from '@bentoproject/accordion';
...
 ```
 
-### Example: Include via `<script>`
+### Include via `<script>`
+
+```html
+<script type="module" src="https://cdn.ampproject.org/bento.mjs"></script>
+<script type="nomodule" src="https://cdn.ampproject.org/bento.js"></script>
+<script type="module" src="https://cdn.ampproject.org/v0/bento-accordion-1.0.mjs"></script>
+<script type="nomodule"  src="https://cdn.ampproject.org/v0/bento-accordion-1.0.js"></script>
```
Comment (westonruter) [inline extensions/amp-accordion/1.0/README.md:34]: There's an extra space here (and in the other changed files):

```suggestion
<script type="nomodule" src="https://cdn.ampproject.org/v0/bento-accordion-1.0.js"></script>
```

Comment (westonruter) [reply extensions/amp-accordion/1.0/README.md:34]: Also, `nomodule` should be an attribute not a type, right?

```suggestion
<script type="text/javascript" nomodule src="https://cdn.ampproject.org/v0/bento-accordion-1.0.js"></script>
```

File: extensions/amp-accordion/1.0/README.md [hunk 3]
```diff
@@ -25,7 +25,17 @@ import {defineElement as defineBentoAccordion} from '@bentoproject/accordion';
 defineBentoAccordion();
 ```
 
-### Example: Include via `<script>`
+### Include via `<script>`
+
+```html
+<script type="module" src="https://cdn.ampproject.org/bento.mjs"></script>
+<script type="nomodule" src="https://cdn.ampproject.org/bento.js"></script>
```
Comment (westonruter) [inline extensions/amp-accordion/1.0/README.md:32]:
```suggestion
<script type="text/javascript" nomodule src="https://cdn.ampproject.org/bento.js"></script>
```

Comment (sebastianbenz) [reply extensions/amp-accordion/1.0/README.md:32]: Is this really needed?

File: extensions/amp-accordion/1.0/README.md (hunk 2 above)
Comment (sebastianbenz) [reply extensions/amp-accordion/1.0/README.md:34]: Fixed the space. [nomodule is an attribute](https://html.spec.whatwg.org/multipage/scripting.html#script-nomodule-example).

File: extensions/amp-accordion/1.0/README.md (hunk 1 above)
Comment (sebastianbenz) [reply extensions/amp-accordion/1.0/README.md:35]: No idea where that came from...removed.

File: extensions/amp-accordion/1.0/README.md (hunk 3 above)
Comment (sebastianbenz) [reply extensions/amp-accordion/1.0/README.md:32]: ("text/javascript")

**Comment by sebastianbenz** [comment]
@westonruter thanks for the review!

File: extensions/amp-accordion/1.0/README.md (hunk 3 above)
Comment (westonruter) [reply extensions/amp-accordion/1.0/README.md]: No idea where that came from! 😉 

You're right, remove it. It's not needed.

**Review by westonruter — APPROVED** [review]

File: extensions/amp-inline-gallery/1.0/README.md [hunk 4]
```diff
@@ -17,7 +17,17 @@ import {defineElement as defineBentoInlineGallery} from '@bentoproject/inline-ga
...
 
-### Example: Include via `<script>`
+### Include via `<script>`
+
+```html
+<script type="module" src="https://cdn.ampproject.org/bento.mjs"></script>
+<script nomodule src="https://cdn.ampproject.org/bento.js"></script>
+<script type="module" src="https://cdn.ampproject.org/v0/bento-inline-gallery-1.0.mjs"></script>
+<script nomodule src="https://cdn.ampproject.org/v0/bento-inline-gallery-1.0.js"></script>
+<link rel="stylesheet" type="text/css" href="https://cdn.ampproject.org/v0/bento-inline-gallery-1.0.css">
```
Comment (westonruter) [inline extensions/amp-inline-gallery/1.0/README.md:27]: Speaking of which, it seems `type="text/css"` [isn't needed either](https://stackoverflow.com/a/5409146/93579), as it is the default in HTML5 (and browsers didn't require it before either).

```suggestion
<link rel="stylesheet" href="https://cdn.ampproject.org/v0/bento-inline-gallery-1.0.css">
```

**Comment by westonruter** [comment]
Also, what about adding `crossorigin="anonymous"` to all the `script` and `link` tags?

**Comment by sebastianbenz** [comment]
> Also, what about adding crossorigin="anonymous" to all the script and link tags?

Good call! Completely forgot about this. Added

**Comment by sebastianbenz** [comment]
We need to update all code snippets to follow this pattern.

**Pull request merged**